### ISSUE TITLE
Data imputation

### ISSUE THREAD

**Issue by Nathaniel-Haines** [CCS-Lab/easyml#103]
Do we have plans for adding data imputation as an option in easyml? Currently, users must remove all NA cases to conduct analyses 

There are many data imputation packages that may be worth adding so that users can take advantages of best practices for dealing with missing data.

**Comment by youngahn** [comment]
Yes, that will be great. If imputation is used, users should see which imputation method is used and how many missing data are imputed.
